### PR TITLE
modified inside blockage criteria and added debug codes

### PR DESCRIPTION
**Pull request by precisionmoon**
This is a minor change off the public branch.  Inside blockage now includes boundary points.

### CODE REVIEW

File: src/cts/src/HTreeBuilder.cpp [hunk 1]
```diff
@@ -123,6 +123,14 @@ void HTreeBuilder::preSinkClustering(
                                   options_->getSinkBuffer(),
                                   legalCenter.getX() * wireSegmentUnit_,
                                   legalCenter.getY() * wireSegmentUnit_);
+      if (logger_->debugCheck(CTS, "legalizer", 2) && (center != legalCenter)) {
+        // clang-format off
+	debugPrint(logger_, CTS, "legalizer", 2,
+		   "preSinkClustering legalizeOneBuffer {}: {} => {}",
+		   baseName + std::to_string(clusterCount),
+		   center, legalCenter);
+        // clang-format on
```
Comment (eder-matheus) [inline src/cts/src/HTreeBuilder.cpp:132]: Why the `clang-format` comments? Here and in the other debugPrints

File: src/cts/src/HTreeBuilder.cpp [hunk 2]
```diff
@@ -123,6 +123,14 @@ void HTreeBuilder::preSinkClustering(
                                   options_->getSinkBuffer(),
                                   legalCenter.getX() * wireSegmentUnit_,
                                   legalCenter.getY() * wireSegmentUnit_);
+      if (logger_->debugCheck(CTS, "legalizer", 2) && (center != legalCenter)) {
```
Comment (eder-matheus) [inline src/cts/src/HTreeBuilder.cpp:126]: You don't need the `logger_->debugCheck(CTS, "legalizer", 2)` in the if statement. The `debugPrint` function already checks this for you. Here and in the other debugPrints.

**Review by eder-matheus — CHANGES_REQUESTED** [review]

File: src/cts/src/HTreeBuilder.cpp (hunk 1 above)
Comment (precisionmoon) [reply src/cts/src/HTreeBuilder.cpp:132]: clang wants to expand debugPrint line into like 10+ lines.  I want to make debug statements compact so this doesn't impact  readability of the rest of the codes.

File: src/cts/src/HTreeBuilder.cpp (hunk 2 above)
Comment (precisionmoon) [reply src/cts/src/HTreeBuilder.cpp:126]: I want to issue debugPrint only if legalizer moved things (center != legalCenter).  Is there an alternate way without debugCheck?

Comment (eder-matheus) [reply src/cts/src/HTreeBuilder.cpp:126]: You can keep the `center != legalCenter`, but remove the `logger_->debugCheck(CTS, "legalizer", 2)` from the if statement.

Comment (precisionmoon) [reply src/cts/src/HTreeBuilder.cpp]: Good idea.  Done.

**Review by eder-matheus — APPROVED** [review]
@vvbandeira FYI

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**